Mathlib/Data/PNat/Defs.lean

```diff
@@ -33,8 +35,10 @@ notation "ℕ+" => PNat
 instance : One ℕ+ :=
   ⟨⟨1, Nat.zero_lt_one⟩⟩
 
+def val : ℕ+ → ℕ := Subtype.val
```

Comment:
```suggestion
@[coe]
def val : ℕ+ → ℕ := Subtype.val
```

I'm not super confident about these things, but if we add @[coe] here and rebuild, the `[norm_cast]` errors in the Basic file seem to go away!